BB-CMU/BB_Digital_Twin_proxy_in_cloud.md

```diff
@@ -1,17 +1,17 @@
 
-# Digital Twin
+# Digital Twin proxy in cloud
 
 ## BB Tags(s)
 <!-- Tag(s) define in which area(s) (cloud, in-vehicle) the BB is executed, and what type of BB it is (tool, process, microservice) -->
-BB-SC-TC
+BB-MU
```

Comment:
@LS0205 if its a Digital Twin proxy in cloud then BB-CMU?

Reply:
done